LidarMap.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from sensor_msgs.msg import LaserScan
+
+class LidarMap:
+    def __init__(self, laser_scan: LaserScan, size : int = 11, resolution : float = 1):
+        """
+        Initialize the map using a ROS 2 LaserScan scan.
+        :param lidar_scan: A LaserScan message object.
+        :param size: The width and height of the square map in cells.
+        :param resolution: The size of each cell in meters.
+        """
+        self.size = size  # size of the map in cells
+        self.resolution = resolution  # resolution of a cell in meters
+        self.map = np.zeros((size, size), dtype=bool)  # initialize the map with boolean values
+        self.position = (size-1, size // 2)  # point that the lidar is taking scan from
+        
+        self.process_scan(laser_scan)
+
+    def process_scan(self, laser_scan: LaserScan) -> None:
+        """
+        Process the Laser scan and update the map accordingly.
+        :param lidar_scan: A LaserScan message object.
+        """
+        angles = np.arange(laser_scan.angle_min, laser_scan.angle_max + laser_scan.angle_increment, laser_scan.angle_increment)
+        angles = np.radians(angles) # convert degrees to radians
```

Comment:
zbędny komentarz, jak chcecie lepiej opisać to dajcie nazwę zmiennej np `angles_radians` a poprzedniej `angles_degrees`